knowledge_graphs/parse_repo_into_neo4j.py

```
@@ -0,0 +1,280 @@
+"""
+DirectNeo4jExtractor for parsing repositories into Neo4j knowledge graph.
+"""
+import asyncio
+import logging
+from typing import Optional, Dict, Any, List
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)